setup.cfg

```
@@ -27,8 +27,7 @@ classifiers =
 [options]
 packages = find:
 install_requires =
-    audeer >=1.14.0
-    audobject >=0.6.1
+    audobject >=0.7.0
```

We should update once we have a new release of `audobject` that fixes https://github.com/audeering/audobject/pull/46

---

Can now be updated to 
```suggestion
    audobject >=0.7.1
```